BD-3369 LINE update (#7902)
braze-inc/braze-docs: _docs/_user_guide/message_building_by_channel/line/line_setup.md

{: .reset-td-br-1 .resest-td-br-2}

### Required account type

To sync followers into Braze, your LINE account needs to be verified or premium. When you create an account, its default status will be unverified. You'll need to request account verification.

### Applying for a verified LINE account

{% alert important %}
Verified accounts are only available for accounts based in Japan, Taiwan, Thailand, and Indonesia.
{% endalert %}

1. On the LINE **Official Account** page, select **Settings**.
2. Under **Information Disclosure Verification Status**, select **Request Account Verification**.
3. Enter the required information.
4. Wait for a notification with the review results.

If you want to sync users who followed a specific channel before that channel was synced with Braze, ask your customer success manager or account manager to [submit a request](https://servicedesk.braze.com/plugins/servlet/desk/portal/12) to the WhatsApp team.

## Integrating LINE

> Follow these steps to set up consistent user updates, bring over existing users' LINE IDs, and sync them all to LINE’s subscription states:

Reviewer: ```suggestion
To set up consistent user updates, bring over existing users' LINE IDs, and sync them all to LINE’s subscription states:
```

1. [Import or update existing known users](#step-1-import-or-update-existing-line-users)
2. [Integrate the LINE channel](#step-2-integrate-line-channel)
3. [Request subscription status sync](#step-3-request-a-subscription-status-sync)
4. [Update user update methods](#step-4-change-your-user-update-methods)
5. [(Optional) Merge users](#step-5-merge-profiles-optional)

## Step 1: Import or update existing LINE users

> Users can be imported or updated by any of the methods that Braze supports, including the [`/users/track`]({{site.baseurl}}/api/endpoints/user_data/post_user_track/) endpoint, [CSV import]({{site.baseurl}}/user_guide/data_and_analytics/user_data_collection/user_import/#csv-import), or [Cloud Data Ingestion]({{site.baseurl}}/user_guide/data_and_analytics/cloud_ingestion/). 

Reviewer: ```suggestion
You can import or update users using any of the methods that Braze supports, including the [`/users/track`]({{site.baseurl}}/api/endpoints/user_data/post_user_track/) endpoint, [CSV import]({{site.baseurl}}/user_guide/data_and_analytics/user_data_collection/user_import/#csv-import), or [Cloud Data Ingestion]({{site.baseurl}}/user_guide/data_and_analytics/cloud_ingestion/). 
```

Regardless of the method you use, update the `native_line_id` to provide the user’s LINE ID. To learn more the `native_line_id`, see [User setup](#user-setup).

{% alert note %}
The subscription group state shouldn't be specified, and it will be ignored. LINE is the source of truth for user subscription status, which will be synced to Braze either through the subscription sync tool or by event updates.
{% endalert %}

## Step 2: Integrate LINE channel

After your existing LINE user base is imported into or updated in Braze, you can integrate your channel. After integration, new followers of your LINE channel will have unidentified user profiles created when they follow the channel.

> ### Step 2a: Connect your LINE channel to Braze

Reviewer: ```suggestion
### Step 2.1: Connect your LINE channel to Braze
```

![Response settings page with toggles for how your account will handle chats.][2]{: style="max-width:80%;"}

> ### Step 2b: Set up your LINE subscription group in Braze

Reviewer: ```suggestion
### Step 2.2: Set up your LINE subscription group in Braze
```

Here is an example payload to `/users/identify` that targets an unidentified user profile by the user alias `line_id`: 

{% raw %}
```json
{
   "aliases_to_identify": [
       {
           "external_id": "known_external_id_from_your_application",
           "user_alias": {
               "alias_name": "U89f4a626548ccd48482f529a482f138b",
               "alias_label": "line_id"
           }
       }
   ]
}
```
{% endraw %}

If no existing user profile exists for your provided `external_id`, it will be added to the unidentified user profile, making it identified. If a user profile does exist for the `external_id`, all attributes that are exclusively on the unidentified user profile will be copied to the known user profile, including `native_line_id` and the user’s subscription status.

> LINE users known in your application can be updated through the [`/users/track`]({{site.baseurl}}/api/endpoints/user_data/post_user_track/) endpoint by passing their external identifiers and `native_line_id`. If an unidentified user profile already exists for a user and the same `native_line_id` is added to a different user profile through `/users/track`, it will inherit all the subscription states of the unidentified user profile. However, duplicate user profiles will exist with the same `native_line_id`. Any subsequent subscription updates from event updates will update all profiles accordingly. 

Reviewer: ```suggestion
You can update LINE users that are known in your application through the [`/users/track`]({{site.baseurl}}/api/endpoints/user_data/post_user_track/) endpoint by passing their external identifiers and `native_line_id`. If an unidentified user profile already exists for a user and the same `native_line_id` is added to a different user profile through `/users/track`, it will inherit all the subscription states of the unidentified user profile. However, duplicate user profiles will exist with the same `native_line_id`. Any subsequent subscription updates from event updates will update all profiles accordingly. 
```

Here is an example payload to `/users/track` that updates a user profile by the external user ID to add a `native_line_id`: 

{% raw %}
```json
{
   "attributes": [
       {
           "external_id": "known_external_id_from_your_application",
           "native_line_id": "U89f4a626548ccd48482f529a482f138b",
           "other": "attribute"
       }
   ]
}
```
{% endraw %}

## Step 5: Merge profiles (optional)

As described above, there's a possibilty for multiple user profiles to exist with the same `native_line_id`. If your update methods create duplicate user profiles, you can merge unidentified user profiles to identified user profiles with the `/user/merge` endpoint. 

Here's an example payload to `/users/merge` that targets an unidentified user profile by user alias `line_id`:

{% raw %}
```json
{
 "merge_updates": [
   {
     "identifier_to_merge": {
       "user_alias": {
         "alias_name": "U89f4a626548ccd48482f529a482f138b",
         "alias_label": "line_id"
       }
     },
     "identifier_to_keep": {
       "external_id": "known_external_id_from_your_application"
     }
   }
 ]
}
```

> {% endraw %}

Reviewer: ```suggestion
{% endraw %}

{% alert tip %}
To learn more about managing duplicate users in Braze, see [Duplicate Users]({{site.baseurl}}/user_guide/engagement_tools/segments/user_profiles/duplicate_users).
{% endalert %}
```
